.github/workflows/ios-build.yml

```diff
@@ -11,14 +11,15 @@ on:
   push:
     branches:
       - Reanimated2
+
```

Comment:
This file is also outdated (e.g. there are only 4 paths while on `main` there are 8). Please use current version from `main` branch.